qapla/settings.py

```diff
@@ -7,7 +7,7 @@ class SettingsFactory(object):
     """
     ENDPOINTS = {
         'uwsgi': [('local', False)],
-        'tests': [('tests', False)],
+        'tests': [('tests', False), ('local', False)],
```

Comment:
`self.paths` is not used in `SettingsFactory`.

Reply:
Fixed.

Reply:
How about using named tuples here? Docstring is enough though.